lib/spack/spack/test/cmd/list.py

```diff
@@ -84,3 +84,15 @@ def test_list_update(tmpdir, mock_packages):
     assert update_file.exists()
     with update_file.open() as f:
         assert f.read() == "empty\n"
+<<<<<<< HEAD
+
+
+def test_tag1(mock_packages):
+=======
+>>>>>>> 5776319f7536add81ac14c940585ee410ca3257e
+
+
+def test_tag1(parser, specs):
+    output = list("--tag", "tag1")
+    assert "mpich" in output
+    assert "mpich2" in output
```

Comment:
We should also test that other packages are not included:
```python
assert "openmpi" not in output
```

Reply:
Yupp looking into it asap